stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -54,18 +61,30 @@ const SecuritiesTable = ({ location }) => {
       });
   };
 
+  const patchSecurityHandler = (securityId, updates) => {
```

Comment:
I think the name of this function should be changed. Functions/methods that do things should be named almost like commands, "do this", "run that", "update this" rather than being something like "something handler". In this case I would  name the function something like `updateSecurity` or `patchSecurity` (which doesn't conflict with your service method since you call it on an object) to follow your existing naming convention.

Reply:
Okay I've renamed it to patch security